ExampleMod/Content/CustomModType/ModVictoryPose.cs

```diff
@@ -0,0 +1,83 @@
+		public int PoseTime { get; set; } = 60;
+
+		// Since this implements ILocalizedModType, all keys from this ModType will default to using Mods.ModName.VictoryPoses.ClassName.KeyName. This should be unique to avoid conflicts with other mods
+		public virtual string LocalizationCategory => "VictoryPoses";
+		public virtual LocalizedText VictoryCheer => this.GetLocalization(nameof(VictoryCheer), PrettyPrintName);
+
+		// When making a custom ModType, we override and seal Register.
+		// In Register, we do all the "plumbing code" needed to load and register the content
+		protected sealed override void Register() {
+			ModTypeLookup<ModVictoryPose>.Register(this);
+			Type = VictoryPoseLoader.Add(this);
+		}
+
+		// When making a custom ModType, we override and seal SetupContent.
+		// In SetupContent, we do all the "plumbing code" needed to initialize and setup the content
+		public sealed override void SetupContent() {
+			ModContent.Request<Texture2D>(Texture); // Ensure that the texture exists. Doing this here means that the mod won't load rather than crash in-game.
+			_ = VictoryCheer; // By calling this here, we ensure that the localization key is populated into the localization files.
+			VictoryPoseID.Search.Add(FullName, Type); // Populate the Search IdDictionary
+			SetStaticDefaults(); // Finally, we call SetStaticDefaults, where each ModVictoryPose class will implement their specific logic.
+		}
+
+		// These virtual methods are the "hooks" we provide that other modders can use to customize their ModVictoryPose behaviors.
+		// It is useful to document these methods so that other mods using it can understand what each method does. They can view the documentation by hovering over the method name. Typing "///" in Visual Studio will generate a documentation stub, after that add any information you want to the "summary" section. See https://github.com/tModLoader/tModLoader/wiki/Why-Use-an-IDE#mod-documentation for more information about how to enable documentation support in your mod.
+		// If your mod has a GitHub wiki, you might want to also document these methods there as well as an additional reference.
+
+		/// <summary>
+		/// Called when the pose starts for the given player.
+		/// </summary>
+		public virtual void OnStartPose(Player player) {
+		}
+
+		/// <summary>
+		/// Called each game update while the pose is active.
+		/// </summary>
+		public virtual void Update(Player player) {
+		}
+
+		/// <summary>
+		/// Called when the pose is ending.
+		/// </summary>
+		public virtual void OnEndPose(Player player) {
+		}
+
+		/// <summary>
+		/// Use to control the frame of the texture to display.
+		/// </summary>
+		public virtual Rectangle? GetTextureFrame(Asset<Texture2D> texture) {
+			return null;
+		}
+
+		/// <inheritdoc cref="VictoryPosePlayer.ElapsedPoseTime"/>
+		public static float ElapsedPoseTime(Player player) => player.GetModPlayer<VictoryPosePlayer>().ElapsedPoseTime;
```

Comment:
Should be in the same place as `GetActivePose` whether you put them both here or in the loader I don't mind.